assets/package-lock.json

```diff
@@ -4,16 +4,18 @@
   "requires": true,
```

Comment:
I believe there is an inconsistency with the files package.json files ;< Haven't you installed dependencies with `npm install`?  
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/10851084/205258664-d6c30f4d-b243-4725-b26f-a59cc36aabc5.png">

Reply:
I have and it worked. I'll check this again, perhaps it breaks after deleting `deps` or sth